dashboard/app/models/user.rb

```diff
@@ -721,6 +721,15 @@ def secret_word_account?
     sections_as_student.any? {|section| section.login_type == Section::LOGIN_TYPE_WORD}
   end
 
+  # True if the account is teacher-managed, is in at least one picture section, and
+  # is not in any non-picture sections
+  def secret_picture_account_only?
+    return false unless teacher_managed_account?
+    all_pictures = sections_as_student.all? {|section| section.login_type == Section::LOGIN_TYPE_PICTURE}
+    any_pictures = sections_as_student.any? {|section| section.login_type == Section::LOGIN_TYPE_PICTURE}
+    all_pictures && any_pictures
```

Comment:
Is the any_pictures only necessary for the case where we have 0 sections? Could this instead just be `all_pictures && sections_as_student.length > 0`?

Reply:
Yes, that should be fine. At this point it might be irrelevant, because I realized that the spec was slightly ambiguous, and I followed the first part of it ("just in picture password sections"), which conflicts with the second part and the examples given, so I'm clarifying the desired behavior now.

Reply:
Ok, clarified, and we're good to go with the current behavior, but I will make this simplification.